### PR TITLE
Display (sub)game icons again

### PR DESCRIPTION
**Pull request by SmallJoker**
+ Borders if there's no icon

Before:
Before
After:
After

### CODE REVIEW

**Comment by MT-Modder** [comment]
:+1:

File: builtin/fstk/buttonbar.lua [hunk 1]
```diff
@@ -50,21 +50,17 @@ local function buttonbar_formspec(self)
...
-			borders="false"
+		if self.buttons[i].image ~= "" then
+			borders = "false"
 		end
-			
 		formspec = formspec ..
 			string.format("image_button[%f,%f;%f,%f;%s;%s;%s;true;%s]tooltip[%s;%s]",
 					btn_pos.x, btn_pos.y, self.btn_size, self.btn_size,
-					self.buttons[i].image, btn_name, self.buttons[i].caption,
+					self.buttons[i].image:gsub("\\\\", "/"), btn_name, self.buttons[i].caption,
```
Comment (ShadowNinja) [inline builtin/fstk/buttonbar.lua]: This should be using `minetest.formspec_escape`.

Comment (SmallJoker) [inline builtin/fstk/buttonbar.lua]: That has already been done earlier.
https://github.com/minetest/minetest/blob/master/builtin/mainmenu/tab_singleplayer.lua#L60
An example file path of `self.buttons[i].image`:
\\Minetest\\games\\my_game\\menu\\icon.png

I tried adding the drive letter in front of that but it did not solve the problem.

**Comment by SmallJoker** [comment]
Found the source of this bug.
Commit 3c3887bb1 thanks to kahrl for pointing to it.

**Comment by ShadowNinja** [comment]
Should be fixed by 77e20a0c2188ab260e571a0406959113faa3e850.

**Comment by SmallJoker** [comment]
Not completely. 
If there's no icon available for the subgame, the button does not contain a border.
See builtin/fstk/buttonbar.lua

**Pull request closed without merging**